### PR TITLE
updated create/delete steps

### PR DESCRIPTION
**Pull request by kumuduh**
Added operator-roles and oidc-provider create/delete steps

### CODE REVIEW

File: docs/quickstart-rosa.md [hunk 1]
```diff
@@ -191,7 +191,15 @@ ROSA can be installed using command line parameters or in interactive mode.  For
   Pod CIDR [10.128.0.0/14]:
   Host prefix [23]:
   Private cluster (y/N):
-  ```  
+  ```
+  
+  You must create operator-roles and oidc-provider as shown in the above create cluster command's output. For e.g.
```
Comment (paulczar) [inline docs/quickstart-rosa.md:196]: this shouldn't actually be necessary, I think we might need to update the create command to include `--mode auto` which should do these steps for us.

File: docs/quickstart-rosa.md [hunk 2]
```diff
@@ -226,6 +234,12 @@ Deleting a ROSA cluster consists of two parts
   ```bash
   rosa delete cluster --cluster=<cluster-name>
   ```
+   Delete Cluster's operator-roles and oidc-provider as shown in the above delete cluster command's output. For e.g.
+  
+  ```bash
+  rosa delete operator-roles -c <cluster-name>
```
Comment (paulczar) [inline docs/quickstart-rosa.md:240]: this is needed though :)

Comment (kumuduh) [reply docs/quickstart-rosa.md:240]: Changes are in [kumuduh-patch-6](https://github.com/rh-mobb/documentation/tree/kumuduh-patch-6). Thanks !

**Comment by kumuduh** [comment]
propose changes are in new pull request. [kumuduh-patch-6](https://github.com/rh-mobb/documentation/tree/kumuduh-patch-6)

**Pull request closed without merging**